verify-spelling.sh

```diff
@@ -20,8 +20,13 @@ set -o pipefail
 
 TOOL_VERSION="v0.3.4"
 
+# The csi-release-tools directory.
+TOOLS="$(dirname "${BASH_SOURCE[0]}")"
+
+# Directory to check. Default is the parent of the tools themselves.
+ROOT="${1:-${TOOLS}/..}"
+
 # cd to the root path
-ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)"
```

Comment:
This cd is redundant. There is another `cd "${ROOT}"` further down before the actual check.

Reply:
Now I've fixed this with the style similar to that in `verify-shellcheck.sh`.

Reply:
My point was that the following lines can be removed completely:
```
# cd to the root path
cd "${ROOT}"
```

Reply:
For the sake of clarity it would be good to move the "cd" below into the "check spelling" section:

```
# check spelling
RES=0
echo "Checking spelling..."
ERROR_LOG="${TMP_DIR}/errors.log"
cd "$ROOT"
```

Then it is obvious that `git ls-files` is meant to operate on files in that directory.

Reply:
Sorry, I misunderstood! It's fixed now.